package.json

```diff
@@ -1,8 +1,50 @@
 {
-  "name": "icons",
+  "name": "@grafana/icons",
   "version": "1.0.0",
   "description": "Icons for Grafana",
   "repository": "https://github.com/grafana/icons.git",
   "author": "Grafana Labs",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "scripts": {
+    "icons:clean": "rm -rf ./src/icons",
+    "icons:create": "npx @svgr/cli ./svg",
+    "build": "tsc -p ./tsconfig.build.json && rollup -c rollup.config.ts --configPlugin typescript"
```

Comment:
Do you need to run `tsc` here since rollup already handles the typescript compilation?